src/main/java/org/example/NotebooksPage.java

```diff
@@ -0,0 +1,32 @@
+package org.example;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+import java.util.List;
+
+public class NotebooksPage {
+    WebDriver driver;
+
+    public NotebooksPage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public boolean isClickable() {
```

Comment:
What is clickable ? Please change the method name to something more meaningful, e.g. isTheLastElementClickable